clients/src/test/java/org/apache/kafka/common/requests/OffsetFetchRequestTest.java

```diff
@@ -55,6 +55,7 @@ public void setUp() {
                                    new TopicPartition(topicTwo, partitionTwo));
         builder = new OffsetFetchRequest.Builder(
             groupId,
+            false,
```

Comment:
For my own education: why do we set `false` here?

Reply:
Just for the purpose of having more case coverage, the default builder is used in other unit tests and we already have a unit test which sets the flag to true.